### PR TITLE
go.mod: Update packages to latest

### PR DESCRIPTION
**Pull request by prarit**
Update the dependencies to their latest versions.

TODO: tview & carapace packages require a bit of investigation.  I will
ping rsteube for assistance with the carapace changes.

Signed-off-by: Prarit Bhargava <prarit@redhat.com>

### CODE REVIEW

**Comment by prarit** [comment]
@rsteube , could you update to 'github.com/rsteube/carapace v0.1.5' and see if you can fix the compile warnings? I wasn't able to make much headway with them.

**Review by bmeneguele — COMMENTED** [review]
Nice! Thanks for that @prarit :))
I'll follow the changes and review once it's completed.

**Comment by rsteube** [comment]
tried that a while ago, but got stuck with some small issues and the prioritised sth. else: https://github.com/rsteube/lab/tree/update-carapace
will have a look

**Comment by rsteube** [comment]
should be ok now: https://github.com/rsteube/lab/commit/d864009a85df6f213f88685834dbea317e23c7d8

**Comment by prarit** [comment]
> @prarit should be ok now: [rsteube@d864009](https://github.com/rsteube/lab/commit/d864009a85df6f213f88685834dbea317e23c7d8)

Thanks :+1:

**Review by rsteube — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -1,27 +1,60 @@
...
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/viper v1.7.1
+	github.com/stretchr/testify v1.6.1
 	github.com/tcnksm/go-gitconfig v0.1.2
-	github.com/xanzy/go-gitlab v0.33.1-0.20200713191942-71ea998bed24
-	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
+	github.com/xanzy/go-gitlab v0.39.0
```
Comment (bmeneguele) [inline go.mod:47]: Nice. With this update, MR #450 may get concluded.

**Comment by bmeneguele** [comment]
@prarit, the changes looks good and are working just fine.
But it seems you need to update some tests code: `Test_mrCmd_MR_description_and_options` is prompting a messy return and `Test_mrShow` seems to have additional whitespace somewhere.

**Review by bmeneguele — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (prarit) [reply go.mod:47]: @bmeneguele can you test #450 on top of my changes?  Or do you just want me to merge and then you can deal with #450?

Comment (bmeneguele) [reply go.mod:47]: @prarit you can merge this one and I deal with #450 later.

**Pull request merged**